data/blank_mod/common/dynamic_modifiers/aat_dynamic_modifiers.txt

```diff
@@ -0,0 +1,1240 @@
+	air_cas_efficiency = FIN_finnish_airforce_air_cas_efficiency #0.1
+	air_superiority_efficiency = FIN_finnish_airforce_air_superiority_efficiency #0.1
+}
+
+FIN_finnish_navy = {
+	enable = { always = yes }
+
+	icon = GFX_idea_FIN_finnish_navy #Needs to match any hidden NS associated to this, such as FIN_hidden_dd_and_coastal_defense_production_ns and FIN_hidden_dd_and_coastal_defense_production_ns_no_mtg
+
+	experience_gain_navy_factor = FIN_finnish_navy_experience_gain_navy_factor #0.05
+	mines_planting_by_fleets_factor = FIN_finnish_navy_mines_planting_by_fleets_factor #0.05
+	navy_anti_air_attack_factor = FIN_finnish_navy_navy_anti_air_attack_factor #0.05
+	navy_spirit_category_type_cost_factor = FIN_finnish_navy_navy_spirit_category_type_cost_factor #-0.1
+
+	#LATER MODIFIERS
+	custom_modifier_tooltip = FIN_finnish_navy_dynamic_modifier_tt #Bonuses applied via hidden NS (FIN_hidden_cas_production_ns, FIN_hidden_fighter_production_ns, FIN_hidden_naval_bomber_production_ns)
+	convoy_retreat_speed = FIN_finnish_navy_convoy_retreat_speed #0.1
+	refit_speed = FIN_finnish_navy_refit_speed #0.1
+	spotting_chance = FIN_finnish_navy_spotting_chance #0.05
+	repair_speed_factor = FIN_finnish_navy_repair_speed_factor #0.1
+	production_speed_dockyard_factor = FIN_finnish_navy_production_speed_dockyard_factor #0.1
+	industrial_capacity_dockyard = FIN_finnish_navy_industrial_capacity_dockyard #0.1
+	extra_marine_supply_grace = FIN_finnish_navy_extra_marine_supply_grace #48
+	naval_invasion_capacity = FIN_finnish_navy_naval_invasion_capacity #15
+	naval_mines_effect_reduction = FIN_finnish_navy_naval_mines_effect_reduction #0.15
+	experience_gain_navy_unit_factor = FIN_finnish_navy_experience_gain_navy_unit_factor #0.1
+	naval_coordination = FIN_finnish_navy_naval_coordination #0.05
+	naval_equipment_upgrade_xp_cost = FIN_finnish_navy_naval_equipment_upgrade_xp_cost #-0.1
+}
+
+FIN_fascist_regime = {
+	enable = { always = yes }
+
+	icon = GFX_idea_FIN_a_fascist_regime
+
+	war_support_factor = FIN_fascist_regime_war_support_factor #0.05
+	send_volunteer_size = FIN_fascist_regime_send_volunteer_size #3
+
+	#LATER MODIFIERS
+	army_org_factor = FIN_fascist_regime_army_org_factor #0.05
+	max_command_power = FIN_fascist_regime_max_command_power #25
+	special_forces_cap = FIN_fascist_regime_special_forces_cap #0.3
+	special_forces_no_supply_grace = FIN_fascist_regime_special_forces_no_supply_grace #24
+	special_forces_out_of_supply_factor = FIN_fascist_regime_special_forces_out_of_supply_factor #-0.15
+	special_forces_attack_factor = FIN_fascist_regime_special_forces_attack_factor #0.10
+	special_forces_training_time_factor = FIN_fascist_regime_special_forces_training_time_factor #0.2
+	justify_war_goal_time = FIN_fascist_regime_justify_war_goal_time # -0.35
+	military_industrial_organization_research_bonus = FIN_fascist_regime_military_industrial_organization_research_bonus #0.1
+	consumer_goods_factor = FIN_fasicst_regime_consumer_goods_factor #-0.1
```

Comment:
Is this misspelling from the base game? Should we be concerned?

Reply:
Just a misspelled variable name, won't cause any issues.